spec/factories.rb

```diff
@@ -45,7 +46,7 @@
   end
   factory :inspection do
     status "Passed"
```

Comment:
This also shows that status is inadequate. In fact, this is an invalid status, not one from the list.